docs/docs-components/handleCodeSandbox.js

```diff
@@ -61,11 +61,8 @@ const handleCodeSandbox = async ({ code, title }: {| code: string, title: string
           dependencies: {
             react: 'latest',
             'react-dom': 'latest',
-            // $FlowIssue[exponential-spread]
-            ...(baseComponents.length > 0 ? { gestalt: 'latest' } : {}),
-            ...(additionalComponents.includes('DatePicker')
-              ? { 'gestalt-datepicker': 'latest' }
-              : {}),
+            gestalt: 'latest',
+            'gestalt-datepicker': 'latest',
```

Comment:
Thanks for fixing this. 🙂 I'm curious if including `gestalt-datepicker` for all sandboxes will increase loading times for those not using DatePicker? We may need to re-add that logic to ensure we only load this package when necessary